README.md

```
@@ -34,188 +35,28 @@ composer require --dev drupal/testmode
 
 ## How it works
-1. When writing Behat tests, all test content items (nodes,
-   terms, users) follow specific pattern. For example, node titles start with
-   `[TEST] `.
-2. A machine name of a view, which needs to be tested, is added to
-   Testmode configuration form.
-3. Behat test tagged with `@testmode` will put
-   the site in test mode that will filter-out all items in the view that do not
-   fit the pattern, leaving only content items created by the test.
+
+1. When writing Behat tests, all test content items (nodes, terms, users) follow specific pattern. For example, node titles start with `[TEST] `.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)